lib/generators/administrate/routes/routes_generator.rb

```diff
@@ -63,6 +58,26 @@ def namespaced_models
         database_models.select { |model| model.to_s.include?("::") }
       end
 
+      def generate_resource_routes(resource)
+        if resource.include?("/")
+          parts = resource.split("/")
+          generate_nested_resource_routes(parts, 0)
```

Comment:
What do you think about the number here (and below, in the implementation) being constants? This should help us avoid having some magic numbers spread about this file.